R/whatNWISData.r

```diff
@@ -57,52 +50,48 @@
 #' @importFrom lubridate parse_date_time
 #' @examples
 #' \dontrun{
-#' availableData <- whatNWISdata('05114000')
+#' availableData <- whatNWISdata(siteNumber = '05114000')
 #' # To find just unit value ('instantaneous') data:
-#' uvData <- whatNWISdata('05114000',service="uv")
-#' uvDataMulti <- whatNWISdata(c('05114000','09423350'),service=c("uv","dv"))
-#' site_ids <- c("01491000","01645000")
-#' flowAndTemp <- whatNWISdata(site_ids, parameterCd=c("00060","00010"))
+#' uvData <- whatNWISdata(siteNumber = '05114000',service="uv")
+#' uvDataMulti <- whatNWISdata(siteNumber = c('05114000','09423350'),service=c("uv","dv"))
+#' flowAndTemp <- whatNWISdata(stateCd = "WI", service = "uv", parameterCd = c("00060","00010"))
 #' }
-whatNWISdata <- function(siteNumbers,service="all",parameterCd="all",statCd="all"){
+whatNWISdata <- function(...){
```

Comment:
What will break people's code is if they don't name their arguments now. I'm willing to take that chance for a far more useful function. They could still use:
`siteNumbers,service="all",parameterCd="all",statCd="all"` and that will work (the magic of `dots`)

So really, the script that will break is `whatNWISdata ('01234567')` because it doesn't know that's suppose to be siteNumber, but in exchange we get the full flexibility of dots.